source/customizations/profiles.inc

```diff
@@ -23,13 +23,12 @@ logic will aggregate all profiles into a single consolidated object.
       config_property: config_value
 
 OnDemand uses an inheritance and override approach for profiles.
-The root configuration provides default properties that all profiles inherit.
-Then, profile properties can override these values or define new ones as needed.
+The root profile, this is the default profile created by the system based on all the properties defined outside any profile definition,
+provide the default values that all profiles inherit. Then, profile properties can override these values or define new ones as required.
```

Comment:
I almost liked the first sentence better. Mabye break this up into 2 sentences.

Maybe something like this?
```suggestion
The root profile is the default profile created by the system based on all the properties defined outside any profile definition.  All other profiles will inherit these values.

Then, profile properties can override these values or define new ones as required.
```

Reply:
Fixed